nengo_viz/static/viz_netgraph.js

```diff
@@ -104,6 +131,7 @@ VIZ.NetGraph = function(parent, args) {
             self.offsetY = (self.offsetY + yy) / scale - yy;
             
             self.scale = scale * self.scale;
+            interact.margin(30*self.scale);
```

Comment:
> I'm more than happy to take that out. It was an experiment.  The downside of removing it is that corner drags disappear for scaled up ensembles, but that might be less annoying.

:)  Yay experiments.  :)  I'd also note that for this one, even with interact.margin changing based on global zoom, we also lose corner drags for Ensembles that are just resized to be large.  However, since Ensembles have a fixed aspect ratio anyway, we actually never need a corner drag for them anyway.  So I think switching this to a fixed value makes the most sense.  

The only place I'm a bit worried about it for is the Networks, since corner-drag is useful for them, and this same problem will happen.  I think I'll use that as an argument for rethinking the rounded corner rescaling, but that's a separate experiment and PR. :)